mocks/mocks.py

```diff
@@ -18,15 +18,172 @@
+    if config['hubble_constant'].value < 1:
+        config['hubble_constant'] = config['hubble_constant'] * 100
+    config['hubble_constant'] = ensure_unit(config['hubble_constant'],
+                                            u.km / (u.Mpc * u.s))
+    # Density of matter(Dark + baryonic)
+    try:
+        config['Omega_0'] = kwargs.get('Omega_0',
+                                       config['Omega_0'])
+    except KeyError:
+        config['Omega_0'] = default_cosmology.Om0
+    try:
+        assert config['Omega_0']
+    except AssertionError:
+        config['Omega_0'] = default_cosmology.Om0
+    # Density of baryonic matter.
+    try:
+        config['Omega_b'] = kwargs.get('Omega_b',
+                                       config['Omega_b'])
+    except KeyError:
+        config['Omega_b'] = default_cosmology.Ob0
+    try:
+        assert config['Omega_b']
+    except AssertionError:
+        config['Omega_b'] = default_cosmology.Ob0
+    # Dark Energy density
+    config['Omega_Lambda'] = 1 - config['Omega_0']
+    # Temperature of the CMB at z=0.
+    try:
+        config['T_CMB0'] = kwargs.get('T_CMB0',
+                                      config['T_CMB0'])
+    except KeyError:
+        config['T_CMB0'] = default_cosmology.Tcmb0
+    try:
+        assert config['T_CMB0']
+    except AssertionError:
+        config['T_CMB0'] = default_cosmology.Tcmb0
+    config['T_CMB0'] = ensure_unit(config['T_CMB0'],
+                                   u.K)
+    # Effective number of Neutrino species.
+    try:
+        config['Neff'] = kwargs.get('Neff',
+                                    config['Neff'])
+    except KeyError:
+        config['Neff'] = default_cosmology.Neff
+    try:
+        assert config['Neff']
+    except AssertionError:
+        config['Neff'] = default_cosmology.Neff
+    # Mass of each neutrino species.
+    try:
+        config['m_nu'] = kwargs.get('m_nu',
+                                    config['m_nu'])
+    except KeyError:
+        config['m_nu'] = default_cosmology.m_nu
+    try:
+        assert config['m_nu']
+    except AssertionError:
+        config['m_nu'] = default_cosmology.m_nu
+    config['m_nu'] = ensure_unit(config['m_nu'],
+                                 u.eV)
+    # Redshift
+    try:
+        config['redshift'] = kwargs.get('redshift',
+                                        config['redshift'])
+    except KeyError:
+        config['redshift'] = 0
+    if config['redshift'] is None:
+        print(33)
+        config['redshift'] = 0
+
+    return config
 
 
 def scale_light_by_distance(particle_positions,
                             particle_values,
-                            physical_distance=10 * u.Mpc,
-                            target_galaxy_comoving_depth=1 * u.Mpc,
-                            viewing_axis='z',
-                            config_location='config_example.yaml'):
+                            config,
+                            **kwargs):
```

Comment:
Don't need `**kwargs` now. Overriding config values should now be done in the call to `parse_config()`, and _only_ in `parse_config()`. Otherwise there's a risk of accidentally using inconsistent config values at different stages in the calculation.